.github/workflows/ci_pipeline.yml

```diff
@@ -0,0 +1,36 @@
+name: ci-pipeline
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.13'
+
+    - name: checkout
+      run: |
+        cd /home/runner/go
+        mkdir src/k8s.io -p
+        cd /home/runner/go/src/k8s.io
+        git clone https://github.com/kubernetes/kube-openapi.git
```

Comment:
This is checking out the master branch, and not the pull-request. So it's not testing that the pull-request works, but that master works. Unfortunately, once master is broken, we can't merge pull-requests anymore :-(

Can we use this instead: https://github.com/actions/checkout ?

Reply:
@apelisse @dims
I am working on using  https://github.com/actions/checkout .
Reason for not using was #225